### PR TITLE
fix clippy warnings in newer versions

### PR DESCRIPTION
**Pull request by cssivision**
Refs: https://github.com/tokio-rs/tokio/issues/3585

### CODE REVIEW

File: tokio/src/runtime/task/state.rs [hunk 1]
```diff
@@ -29,13 +29,13 @@ const LIFECYCLE_MASK: usize = 0b11;
...
-const JOIN_INTEREST: usize = 0b1_000;
+const JOIN_INTEREST: usize = 0b1000;
 
 /// A join handle waker has been set
-const JOIN_WAKER: usize = 0b10_000;
+const JOIN_WAKER: usize = 0b1_0000;
 
 /// The task has been forcibly cancelled.
-const CANCELLED: usize = 0b100_000;
+const CANCELLED: usize = 0b10_0000;
```
Comment (Darksonn) [inline tokio/src/runtime/task/state.rs:38]: I have a suspicion that these underscores were set here deliberately.

**Review by Darksonn — COMMENTED** [review]
I have no comments on the remaining things.

File: tokio/src/runtime/task/state.rs (hunk 1 above)
Comment (cssivision) [reply tokio/src/runtime/task/state.rs:38]: see https://rust-lang.github.io/rust-clippy/master/index.html#unusual_byte_groupings

feel free to revert it.

**Review by Darksonn — APPROVED** [review]
Thanks

**Pull request merged**